static/js/components/NewGroupForm.jsx

```diff
@@ -1,78 +1,146 @@
-import React, { useState } from "react";
-import { useDispatch } from "react-redux";
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
 
 import Box from "@material-ui/core/Box";
 import Button from "@material-ui/core/Button";
-import * as Action from "../ducks/groups";
+import Select from "@material-ui/core/Select";
+import Chip from "@material-ui/core/Chip";
+import MenuItem from "@material-ui/core/MenuItem";
+import Input from "@material-ui/core/Input";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+
+import * as groupsActions from "../ducks/groups";
+import * as usersActions from "../ducks/users";
 import styles from "./NewGroupForm.css";
 
+const getStyles = (userID, userIDs = [], theme) => ({
+  fontWeight:
+    userIDs.indexOf(userID) === -1
+      ? theme.typography.fontWeightRegular
+      : theme.typography.fontWeightMedium,
+});
+
 const NewGroupForm = () => {
   const dispatch = useDispatch();
+  const { allUsers } = useSelector((state) => state.users);
+
   const [formState, setState] = useState({
     name: "",
     group_admins: [],
   });
 
-  const handleSubmit = (event) => {
+  useEffect(() => {
+    if (allUsers.length === 0) {
+      dispatch(usersActions.fetchUsers());
+    }
+  }, [dispatch, allUsers]);
+
+  const userIDToName = {};
+  allUsers.forEach((u) => {
+    userIDToName[u.id] = u.username;
+  });
+
+  const handleSubmit = async (event) => {
     event.preventDefault();
-    dispatch(Action.addNewGroup(formState));
-    setState({
-      name: "",
-      group_admins: [],
-    });
+    const result = await dispatch(groupsActions.addNewGroup(formState));
+    if (result.status === "success") {
+      setState({
+        name: "",
+        group_admins: [],
+      });
+    }
   };
 
   const handleChange = (event) => {
     const newState = {};
-    newState[event.target.name] =
-      event.target.name === "groupAdmins"
-        ? event.target.value.split(",")
-        : event.target.value;
+    newState[event.target.name] = event.target.value;
     setState({
       ...formState,
       ...newState,
     });
   };
 
+  const useStyles = makeStyles((theme) => ({
+    formControl: {
+      margin: theme.spacing(1),
```

Comment:
```suggestion
      margin: `${theme.spacing(1)}px 0`,
```
I think we should only have the top/bottom margins here so that the admin select stays aligned with the group name input. Change results in:
![Screenshot 2020-09-16 153248](https://user-images.githubusercontent.com/17696889/93399707-f757c880-f832-11ea-8533-be9553dfb0c5.png)

vs right now:
![Screenshot 2020-09-16 154117](https://user-images.githubusercontent.com/17696889/93399776-15bdc400-f833-11ea-921b-261b964574f9.png)